### DISCUSSION TITLE
Drag and drop?

### DISCUSSION THREAD

**Discussion by SamSandq** [frang75/nappgui_src#88] [Ideas]
I am writing a photo management app, with a collection of thumbnails of photos as well as a list (to be in the forthcoming treeview). Now, I wish to be able grab a thumbnail of a photo in the grid of photos and move or copy it to the list, dropping it on another collection.

Effectively drag and drop.

How to achieve this in NappGUI?

**Comment by chris-deep** [comment]
Curious too, this is the search I ran to look for drag...  Looking forward to know this as well.

[https://github.com/search?q=repo%3Afrang75%2Fnappgui_src%20drag&type=code](url)

**Comment by frang75** [comment]
Hi @SamSandq, @chris-deep. 
At the moment Drag and Drop support is not implemented. I open a Discord channel to plan this feature.
Sam, email me with your Discord user and I will add you to the server.

Also referenced in https://github.com/frang75/nappgui_src/issues/39

**Comment by SamSandq** [comment]
HI,

I just signed up in order to participate.

Username should be

samsandqvist


---------------------------------
Sam


Den mån 15 jan. 2024 kl 06:24 skrev Francisco García Collado <
***@***.***>:
